### ISSUE TITLE
Current E2E Tests - Tracking Issue

### ISSUE THREAD

**Issue by geriux** [wordpress-mobile/gutenberg-mobile#5325]
This is a ticket to keep track of the current E2E tests, the ones that are disabled or any updates related to them.

# Current E2E tests

## Canaries E2E tests

- Gutenberg Editor File Block tests
- Gutenberg Editor Heading tests
- Gutenberg Editor Image Block tests
- Gutenberg Editor Slash Inserter tests

## Full E2E tests

- Gutenberg Editor Audio Block tests
- Gutenberg Editor tests for Block insertion
- Gutenberg Editor Cover Block test
- Gutenberg Editor Drag & Drop blocks tests
   **Two tests disabled in**: https://github.com/WordPress/gutenberg/pull/46340
- Gutenberg Editor Blocks (Initial HTML) test
- Gutenberg Editor tests for Paragraph Block
- Gutenberg Editor paste tests
   **Some tests disabled in**: https://github.com/WordPress/gutenberg/pull/46340/files
- Gutenberg Editor Device Rotation tests
- Gutenberg Editor Search Block tests
- Gutenberg Editor Unsupported Block Editor Tests
   **Disabled for Android in**: https://github.com/WordPress/gutenberg/pull/46542

**Comment by fluiddot** [comment]
In a second iteration of the list, it would be great to identify the ones that could be replaced by an integration test. If an E2E test doesn't really test a native functionality, we should drop it in favor of an integration test.

**Comment by geriux** [comment]
I've gone through the list and I've made some updates in: https://github.com/WordPress/gutenberg/pull/46583

- Gutenberg Editor Columns Block test **(Removed)**
   This test was removed since it already has coverage in integration tests ([see](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/columns/test/edit.native.js))
- Gutenberg Editor Latest Post Block tests **(Removed)**
   It was migrated to an integration test (**see**)
- Gutenberg Editor tests for List block, Gutenberg Editor tests for List block (end), Gutenberg Editor tests for List block **(Removed)**
   These tests were removed since they were disabled when the new version of the List block `V2` was merged, now it uses Inner Blocks so we don't have to test the List implementation with Aztec since is no longer used, we added plenty of integration tests for coverage ([see](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/list/test/edit.native.js))
- Gutenberg Editor More Block test **(Removed)**
   This test was migrated into an integration test (**see**)
- Gutenberg Editor Verse Block Tests **(Removed)**
   This test was moved to their integration tests ([see](https://github.com/WordPress/gutenberg/pull/46583/commits/0a2c0e7c0d673101a1b403501a0e4738e9d33f07#diff-acc781d1d1cf1d5c7ade09c14a42453650f7f7e23184ba9ea1f8463cd22780ba))
- Gutenberg Editor Separator Block test **(Removed)**
   This test was migrated into an integration test ([see](https://github.com/WordPress/gutenberg/pull/46583/commits/d9ee9f602901aa846a97bb96046e2fd94c1bcb38#diff-ca67391ca09de8fe01c19097c7b9902e8acd51a8804de75c545fa516c8bb0a25))

~I will update the original list once the PR referenced above is merged.~ Updated.

**Comment by geriux** [comment]
**Gutenberg Editor tests for Block insertion 2** will be updated in https://github.com/WordPress/gutenberg/pull/46822 to remove the `inserts block before selected block` test due to flakiness, now it will be covered by integration tests. I'll update this comment once that PR is merged.

It is now merged ✅

**Comment by geriux** [comment]
As a follow-up of https://github.com/WordPress/gutenberg/pull/46822 I created another PR https://github.com/WordPress/gutenberg/pull/46882 which will remove `Gutenberg Editor tests for Block insertion 2` entirely in favor of integration tests, since it doesn't test native features. ~I will update the main ticket once it gets merged.~

**Gutenberg Editor tests for Block insertion 2** were finally removed in https://github.com/WordPress/gutenberg/pull/46882

**Comment by geriux** [comment]
This tracking issue hasn't been updated since January, after some discussions in https://github.com/WordPress/gutenberg/pull/55541#pullrequestreview-1692565208 which removes other tests that are currently disabled, I think it's best to close this. We can always open new GitHub issues for specific tests if needed.
